config/della/required_modules.txt

```diff
@@ -0,0 +1,2 @@
+openmpi/gcc/4.1.2
+anaconda3/2022.10
```

Comment:
If you create a `module_init` file [like this](https://github.com/simonsobs/soconda/blob/main/config/perlmutter/module_init), then that snippet will be pasted into the final soconda module file.  The comment / command syntax should follow either Lua or TCL syntax, depending on whether your system is using `lmod` or traditional `modules`.  So any module commands in that file will get executed before loading the stack.  Just in case it helps with your system setup.

Reply:
I'll take a look and come back to you. Thank you for the suggestion